[bug] Error related to smoke detector CC Notification

**Checklist:**

[x] I am **not** using HomeAssistant.
[x] I have checked the troubleshooting section and my problem is **not** described there.
[x] I have read the changelog problem was **not** mentioned there.

Build/Run method

[ ] Docker
[ ] PKG
[ ] Snap package
[x] Manually built (git clone - npm install - npm run build )

zwavejs2mqtt version: 7.0.0
zwave-js version: 3.0.1

### Describe the bug

I am getting regular restarts which seem to be connected to my smoke detector nodes. The JSON esport of the node in question is here:

```json
{"id":49,"deviceId":"312-3-1","manufacturer":"First Alert (BRK Brands Inc)","manufacturerId":312,"productType":1,"productId":3,"name":"Smoke","loc":"Bath","values":[{"id":"49-112-0-1","nodeId":49,"commandClass":112,"commandClassName":"Configuration","endpoint":0,"property":1,"propertyName":"SuperVision report timeout","type":"number","readable":true,"writeable":true,"description":"ZCOMBO will send the message over SuperVision Command Class and it will wait for the SuperVision report from the Controller for the SuperVision report timeout time.","label":"SuperVision report timeout","default":1500,"stateless":false,"min":500,"max":5000,"unit":"seconds","list":false,"value":1500,"lastUpdate":1616736002749,"newValue":1500},{"id":"49-112-0-2","nodeId":49,"commandClass":112,"commandClassName":"Configuration","endpoint":0,"property":2,"propertyName":"SuperVision retry count","type":"number","readable":true,"writeable":true,"description":"If the SuperVision report is not received within the SuperVision report timeout time, the ZCOMBO will retry sending the message again. Upon exceeding the max retry, the ZCOMBO device will send the next message available in the queue.","label":"SuperVision retry count","default":1,"stateless":false,"min":0,"max":5,"list":false,"value":1,"lastUpdate":1616736002750,"newValue":1},{"id":"49-112-0-3","nodeId":49,"commandClass":112,"commandClassName":"Configuration","endpoint":0,"property":3,"propertyName":"SuperVision wait time","type":"number","readable":true,"writeable":true,"description":"Before retrying the message, ZCOMBO will wait for the SuperVision wait time. Actual wait time is calculated using the formula: Wait Time = SuperVision wait time base-value + random-value + (attempt-count x 5 seconds). The random value will be between 100 and 1100 milliseconds.","label":"SuperVision wait time","default":5,"stateless":false,"min":1,"max":60,"unit":"s","list":false,"value":5,"lastUpdate":1616736002752,"newValue":5},{"id":"49-113-0-Smoke Alarm-Sensor status","nodeId":49,"commandClass":113,"commandClassName":"Notification","endpoint":0,"property":"Smoke Alarm","propertyName":"Smoke Alarm","propertyKey":"Sensor status","propertyKeyName":"Sensor status","type":"number","readable":true,"writeable":false,"label":"Sensor status","stateless":false,"ccSpecific":{"notificationType":1},"min":0,"max":255,"list":true,"states":[{"text":"idle","value":0},{"text":"Smoke detected","value":2}],"value":0,"lastUpdate":1616736002753,"newValue":0},{"id":"49-113-0-Smoke Alarm-Alarm status","nodeId":49,"commandClass":113,"commandClassName":"Notification","endpoint":0,"property":"Smoke Alarm","propertyName":"Smoke Alarm","propertyKey":"Alarm status","propertyKeyName":"Alarm status","type":"number","readable":true,"writeable":false,"label":"Alarm status","stateless":false,"ccSpecific":{"notificationType":1},"min":0,"max":255,"list":true,"states":[{"text":"idle","value":0},{"text":"Smoke alarm test","value":3},{"text":"Alarm silenced","value":6}],"value":0,"lastUpdate":1616736002755,"newValue":0},{"id":"49-113-0-CO Alarm-Sensor status","nodeId":49,"commandClass":113,"commandClassName":"Notification","endpoint":0,"property":"CO Alarm","propertyName":"CO Alarm","propertyKey":"Sensor status","propertyKeyName":"Sensor status","type":"number","readable":true,"writeable":false,"label":"Sensor status","stateless":false,"ccSpecific":{"notificationType":2},"min":0,"max":255,"list":true,"states":[{"text":"idle","value":0},{"text":"Carbon monoxide detected","value":2}],"value":0,"lastUpdate":1616736002756,"newValue":0},{"id":"49-113-0-CO Alarm-Maintenance status","nodeId":49,"commandClass":113,"commandClassName":"Notification","endpoint":0,"property":"CO Alarm","propertyName":"CO Alarm","propertyKey":"Maintenance status","propertyKeyName":"Maintenance status","type":"number","readable":true,"writeable":false,"label":"Maintenance status","stateless":false,"ccSpecific":{"notificationType":2},"min":0,"max":255,"list":true,"states":[{"text":"idle","value":0},{"text":"Replacement required, End-of-life","value":5}],"value":0,"lastUpdate":1616736002758,"newValue":0},{"id":"49-113-0-CO Alarm-Alarm status","nodeId":49,"commandClass":113,"commandClassName":"Notification","endpoint":0,"property":"CO Alarm","propertyName":"CO Alarm","propertyKey":"Alarm status","propertyKeyName":"Alarm status","type":"number","readable":true,"writeable":false,"label":"Alarm status","stateless":false,"ccSpecific":{"notificationType":2},"min":0,"max":255,"list":true,"states":[{"text":"idle","value":0},{"text":"Alarm silenced","value":6}],"value":0,"lastUpdate":1616736002759,"newValue":0},{"id":"49-113-0-System-Hardware status","nodeId":49,"commandClass":113,"commandClassName":"Notification","endpoint":0,"property":"System","propertyName":"System","propertyKey":"Hardware status","propertyKeyName":"Hardware status","type":"number","readable":true,"writeable":false,"label":"Hardware status","stateless":false,"ccSpecific":{"notificationType":9},"min":0,"max":255,"list":true,"states":[{"text":"idle","value":0},{"text":"System hardware failure","value":1}],"value":0,"lastUpdate":1616736002761,"newValue":0},{"id":"49-114-0-manufacturerId","nodeId":49,"commandClass":114,"commandClassName":"Manufacturer Specific","endpoint":0,"property":"manufacturerId","propertyName":"manufacturerId","type":"number","readable":true,"writeable":false,"label":"Manufacturer ID","stateless":false,"min":0,"max":65535,"list":false,"value":312,"lastUpdate":1616736002762,"newValue":312},{"id":"49-114-0-productType","nodeId":49,"commandClass":114,"commandClassName":"Manufacturer Specific","endpoint":0,"property":"productType","propertyName":"productType","type":"number","readable":true,"writeable":false,"label":"Product type","stateless":false,"min":0,"max":65535,"list":false,"value":1,"lastUpdate":1616736002764,"newValue":1},{"id":"49-114-0-productId","nodeId":49,"commandClass":114,"commandClassName":"Manufacturer Specific","endpoint":0,"property":"productId","propertyName":"productId","type":"number","readable":true,"writeable":false,"label":"Product ID","stateless":false,"min":0,"max":65535,"list":false,"value":3,"lastUpdate":1616736002765,"newValue":3},{"id":"49-128-0-level","nodeId":49,"commandClass":128,"commandClassName":"Battery","endpoint":0,"property":"level","propertyName":"level","type":"number","readable":true,"writeable":false,"label":"Battery level","stateless":false,"min":0,"max":100,"unit":"%","list":false,"value":95,"lastUpdate":1616736002767,"newValue":95},{"id":"49-128-0-isLow","nodeId":49,"commandClass":128,"commandClassName":"Battery","endpoint":0,"property":"isLow","propertyName":"isLow","type":"boolean","readable":true,"writeable":false,"label":"Low battery level","stateless":false,"list":false,"value":false,"lastUpdate":1616736002768,"newValue":false},{"id":"49-132-0-wakeUpInterval","nodeId":49,"commandClass":132,"commandClassName":"Wake Up","endpoint":0,"property":"wakeUpInterval","propertyName":"wakeUpInterval","type":"number","readable":false,"writeable":true,"label":"Wake Up interval","default":4200,"stateless":false,"min":4200,"max":4200,"step":0,"list":false,"value":4200,"lastUpdate":1616736002769,"newValue":4200},{"id":"49-132-0-controllerNodeId","nodeId":49,"commandClass":132,"commandClassName":"Wake Up","endpoint":0,"property":"controllerNodeId","propertyName":"controllerNodeId","type":"any","readable":true,"writeable":false,"label":"Node ID of the controller","stateless":false,"list":false,"value":1,"lastUpdate":1616736002771,"newValue":1},{"id":"49-134-0-libraryType","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"libraryType","propertyName":"libraryType","type":"any","readable":true,"writeable":false,"label":"Library type","stateless":false,"list":false,"value":6,"lastUpdate":1616736002772,"newValue":6},{"id":"49-134-0-protocolVersion","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"protocolVersion","propertyName":"protocolVersion","type":"any","readable":true,"writeable":false,"label":"Z-Wave protocol version","stateless":false,"list":false,"value":"6.7","lastUpdate":1616736002773,"newValue":"6.7"},{"id":"49-134-0-firmwareVersions","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"firmwareVersions","propertyName":"firmwareVersions","type":"any","readable":true,"writeable":false,"label":"Z-Wave chip firmware versions","stateless":false,"list":false,"value":["11.0","7.0"],"lastUpdate":1616736002775,"newValue":["11.0","7.0"]},{"id":"49-134-0-hardwareVersion","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"hardwareVersion","propertyName":"hardwareVersion","type":"any","readable":true,"writeable":false,"label":"Z-Wave chip hardware version","stateless":false,"list":false,"value":2,"lastUpdate":1616736002776,"newValue":2},{"id":"49-134-0-sdkVersion","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"sdkVersion","propertyName":"sdkVersion","type":"any","readable":true,"writeable":true,"label":"sdkVersion (property)","stateless":false,"list":false,"value":"6.81.6","lastUpdate":1616736002778,"newValue":"6.81.6"},{"id":"49-134-0-applicationFrameworkAPIVersion","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"applicationFrameworkAPIVersion","propertyName":"applicationFrameworkAPIVersion","type":"any","readable":true,"writeable":true,"label":"applicationFrameworkAPIVersion (property)","stateless":false,"list":false,"value":"4.3.0","lastUpdate":1616736002779,"newValue":"4.3.0"},{"id":"49-134-0-applicationFrameworkBuildNumber","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"applicationFrameworkBuildNumber","propertyName":"applicationFrameworkBuildNumber","type":"any","readable":true,"writeable":true,"label":"applicationFrameworkBuildNumber (property)","stateless":false,"list":false,"value":52445,"lastUpdate":1616736002781,"newValue":52445},{"id":"49-134-0-hostInterfaceVersion","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"hostInterfaceVersion","propertyName":"hostInterfaceVersion","type":"any","readable":true,"writeable":true,"label":"hostInterfaceVersion (property)","stateless":false,"list":false,"value":"unused","lastUpdate":1616736002782,"newValue":"unused"},{"id":"49-134-0-hostInterfaceBuildNumber","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"hostInterfaceBuildNumber","propertyName":"hostInterfaceBuildNumber","type":"any","readable":true,"writeable":true,"label":"hostInterfaceBuildNumber (property)","stateless":false,"list":false,"value":0,"lastUpdate":1616736002783,"newValue":0},{"id":"49-134-0-zWaveProtocolVersion","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"zWaveProtocolVersion","propertyName":"zWaveProtocolVersion","type":"any","readable":true,"writeable":true,"label":"zWaveProtocolVersion (property)","stateless":false,"list":false,"value":"6.7.0","lastUpdate":1616736002785,"newValue":"6.7.0"},{"id":"49-134-0-zWaveProtocolBuildNumber","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"zWaveProtocolBuildNumber","propertyName":"zWaveProtocolBuildNumber","type":"any","readable":true,"writeable":true,"label":"zWaveProtocolBuildNumber (property)","stateless":false,"list":false,"value":97,"lastUpdate":1616736002786,"newValue":97},{"id":"49-134-0-applicationVersion","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"applicationVersion","propertyName":"applicationVersion","type":"any","readable":true,"writeable":true,"label":"applicationVersion (property)","stateless":false,"list":false,"value":"11.0.0","lastUpdate":1616736002788,"newValue":"11.0.0"},{"id":"49-134-0-applicationBuildNumber","nodeId":49,"commandClass":134,"commandClassName":"Version","endpoint":0,"property":"applicationBuildNumber","propertyName":"applicationBuildNumber","type":"any","readable":true,"writeable":true,"label":"applicationBuildNumber (property)","stateless":false,"list":false,"value":0,"lastUpdate":1616736002789,"newValue":0}],"groups":[{"text":"Lifeline","value":1,"maxNodes":1,"isLifeline":true,"multiChannel":true},{"text":"Lifeline","value":1,"maxNodes":1,"isLifeline":true,"multiChannel":true}],"neighbors":[10,11,12,13,14,15,17,18,19,2,20,21,22,23,24,25,26,27,28,29,3,30,31,33,34,35,36,37,4,47,5,51,6,8],"ready":true,"available":true,"hassDevices":{},"failed":false,"lastActive":1616736002790,"firmwareVersion":"11.0","isSecure":false,"keepAwake":false,"maxBaudRate":null,"isRouting":true,"isFrequentListening":false,"isListening":false,"inited":true,"hexId":"0x0138-0x0001-0x0003","dbLink":"https://devices.zwave-js.io/?jumpTo=0x0138:0x0001:0x0003:11.0","productLabel":"ZCOMBO","productDescription":"ZCombo-G Smoke/CO Alarm","protocolVersion":3,"zwavePlusVersion":1,"zwavePlusNodeType":0,"zwavePlusRoleType":6,"endpointsCount":0,"isControllerNode":false,"dataRate":40000,"deviceClass":{"basic":4,"generic":7,"specific":1},"status":"Asleep","interviewStage":"Complete","_name":"Smoke (Bath)"}
```

### To Reproduce

Steps to reproduce the behavior:

1. Start zwavejs2mqtt
2. Wait for a smoke detector to report
3. Watch the logs to see errors

### Expected behavior

I'm not sure how the error can be fixed, but I'll provide the logs below.

NOTE: I'm running zwavejs2mqtt as a system service - I will put below the logs from the time the CC Notification was received through when my system service was restarted by systemd

### Additional context

Logs:
```
Mar 26 00:19:53 zwavejs npm[390]: 2021-03-26 00:19:53.435 INFO ZWAVE: Node 49 CC Notification {
Mar 26 00:19:53 zwavejs npm[390]:   type: 9,
Mar 26 00:19:53 zwavejs npm[390]:   event: 5,
Mar 26 00:19:53 zwavejs npm[390]:   label: 'System',
Mar 26 00:19:53 zwavejs npm[390]:   eventLabel: 'Heartbeat',
Mar 26 00:19:53 zwavejs npm[390]:   parameters: undefined
Mar 26 00:19:53 zwavejs npm[390]: }
Mar 26 00:19:53 zwavejs npm[390]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Mar 26 00:19:53 zwavejs npm[390]: TypeError: Cannot read property 'replace' of undefined
Mar 26 00:19:53 zwavejs npm[390]:     at Object.removeSlash (/home/pi/zwavejs2mqtt/lib/utils.js:143:34)
Mar 26 00:19:53 zwavejs npm[390]:     at Gateway.valueTopic (/home/pi/zwavejs2mqtt/lib/Gateway.js:1116:26)
Mar 26 00:19:53 zwavejs npm[390]:     at Gateway.onNotification (/home/pi/zwavejs2mqtt/lib/Gateway.js:342:22)
Mar 26 00:19:53 zwavejs npm[390]:     at ZwaveClient.emit (events.js:314:20)
Mar 26 00:19:53 zwavejs npm[390]:     at ZwaveClient.EventEmitter.emit (domain.js:483:12)
Mar 26 00:19:53 zwavejs npm[390]:     at ZwaveClient.onNodeNotification (/home/pi/zwavejs2mqtt/lib/ZwaveClient.js:716:8)
Mar 26 00:19:53 zwavejs npm[390]:     at ZWaveNode.emit (events.js:314:20)
Mar 26 00:19:53 zwavejs npm[390]:     at ZWaveNode.EventEmitter.emit (domain.js:483:12)
Mar 26 00:19:53 zwavejs npm[390]:     at ZWaveNode.handleNotificationReport (/home/pi/zwavejs2mqtt/node_modules/zwave-js/src/lib/node/Node.ts:2447:10)
Mar 26 00:19:53 zwavejs npm[390]:     at ZWaveNode.handleCommand (/home/pi/zwavejs2mqtt/node_modules/zwave-js/src/lib/node/Node.ts:1861:16)
Mar 26 00:19:53 zwavejs npm[390]: npm ERR! code ELIFECYCLE
Mar 26 00:19:53 zwavejs npm[390]: npm ERR! errno 1
Mar 26 00:19:53 zwavejs npm[390]: npm ERR! zwavejs2mqtt@3.0.1 start: `node bin/www`
Mar 26 00:19:53 zwavejs npm[390]: npm ERR! Exit status 1
Mar 26 00:19:53 zwavejs npm[390]: npm ERR!
Mar 26 00:19:53 zwavejs npm[390]: npm ERR! Failed at the zwavejs2mqtt@3.0.1 start script.
Mar 26 00:19:53 zwavejs npm[390]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Mar 26 00:19:53 zwavejs npm[390]: npm ERR! A complete log of this run can be found in:
Mar 26 00:19:53 zwavejs npm[390]: npm ERR!     /home/pi/.npm/_logs/2021-03-26T05_19_53_563Z-debug.log
Mar 26 00:19:53 zwavejs systemd[1]: zwavejs2mqtt.service: Main process exited, code=exited, status=1/FAILURE
Mar 26 00:19:53 zwavejs systemd[1]: zwavejs2mqtt.service: Failed with result 'exit-code'.
Mar 26 00:19:53 zwavejs systemd[1]: zwavejs2mqtt.service: Service RestartSec=100ms expired, scheduling restart.
Mar 26 00:19:53 zwavejs systemd[1]: zwavejs2mqtt.service: Scheduled restart job, restart counter is at 1.
Mar 26 00:19:53 zwavejs systemd[1]: Stopped zwavejs2mqtt.
Mar 26 00:19:53 zwavejs systemd[1]: Started zwavejs2mqtt.
```

--- a/lib/ZwaveClient.js
+++ b/lib/ZwaveClient.js
@@ -90,7 +90,7 @@ const ZWAVEJS_LOG_FILE = utils.joinPath(storeDir, `zwavejs_${process.pid}.log`)
  * @param {Socket} socket
  * @returns {import('../types/index.js').ZwaveClient}
  */
-function ZwaveClient (config, socket) {
+function ZwaveClient(config, socket) {
   if (!(this instanceof ZwaveClient)) {
     return new ZwaveClient(config)
   }
@@ -140,7 +140,7 @@ inherits(ZwaveClient, EventEmitter)
 
 // ---------- DRIVER EVENTS -------------------------------------
 
-function driverReady () {
+function driverReady() {
   /*
     Now the controller interview is complete. This means we know which nodes
     are included in the network, but they might not be ready yet.
@@ -194,7 +194,7 @@ function driverReady () {
   logger.info(`Scanning network with homeid: ${homeHex}`)
 }
 
-function driverError (error) {
+function driverError(error) {
   this.error = 'Driver: ' + error.message
   this.status = ZWAVE_STATUS.driverFailed
   updateControllerStatus.call(this, this.error)
@@ -202,7 +202,7 @@ function driverError (error) {
   this.emit('event', eventEmitter.driver, 'driver error', error)
 }
 
-function scanComplete () {
+function scanComplete() {
   this.scanComplete = true
 
   updateControllerStatus.call(this, 'Scan completed')
@@ -219,43 +219,43 @@ function scanComplete () {
 
 // ---------- CONTROLLER EVENTS -------------------------------
 
-function updateControllerStatus (status) {
+function updateControllerStatus(status) {
   logger.info(`Controller status: ${status}`)
   this.cntStatus = status
   this.sendToSocket(socketEvents.controller, status)
 }
 
-function onInclusionStarted (secure) {
+function onInclusionStarted(secure) {
   const message = `${secure ? 'Secure' : 'Non-secure'} inclusion started`
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'inclusion started', secure)
 }
 
-function onExclusionStarted () {
+function onExclusionStarted() {
   const message = 'Exclusion started'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'exclusion started')
 }
 
-function onInclusionStopped () {
+function onInclusionStopped() {
   const message = 'Inclusion stopped'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'inclusion stopped')
 }
 
-function onExclusionStopped () {
+function onExclusionStopped() {
   const message = 'Exclusion stopped'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'exclusion stopped')
 }
 
-function onInclusionFailed () {
+function onInclusionFailed() {
   const message = 'Inclusion failed'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'inclusion failed')
 }
 
-function onExclusionFailed () {
+function onExclusionFailed() {
   const message = 'Exclusion failed'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'exclusion failed')
@@ -266,7 +266,7 @@ function onExclusionFailed () {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeAdded (zwaveNode) {
+function onNodeAdded(zwaveNode) {
   logger.info(`Node ${zwaveNode.id}: added`)
 
   // the driver is ready so this node has been added on fly
@@ -288,7 +288,7 @@ function onNodeAdded (zwaveNode) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeRemoved (zwaveNode) {
+function onNodeRemoved(zwaveNode) {
   logger.info(`Node ${zwaveNode.id}: removed`)
   zwaveNode.removeAllListeners()
 
@@ -307,7 +307,7 @@ function onNodeRemoved (zwaveNode) {
  *
  * @param {ReadonlyMap<number, import('zwave-js').HealNodeStatus>} progress
  */
-function onHealNetworkProgress (progress) {
+function onHealNetworkProgress(progress) {
   const toHeal = [...progress.values()]
   const healedNodes = toHeal.filter(v => v !== 'pending')
   let message
@@ -335,7 +335,7 @@ function onHealNetworkProgress (progress) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeStatus (zwaveNode) {
+function onNodeStatus(zwaveNode) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (node) {
@@ -356,7 +356,7 @@ function onNodeStatus (zwaveNode) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeReady (zwaveNode) {
+function onNodeReady(zwaveNode) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (!node) {
@@ -417,8 +417,7 @@ function onNodeReady (zwaveNode) {
   )
 
   logger.info(
-    `Node ${node.id} ready: ${node.manufacturer} - ${
-      node.productLabel
+    `Node ${node.id} ready: ${node.manufacturer} - ${node.productLabel
     } (${node.productDescription || 'Unknown'})`
   )
 }
@@ -429,7 +428,7 @@ function onNodeReady (zwaveNode) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeInterviewCompleted (zwaveNode) {
+function onNodeInterviewCompleted(zwaveNode) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (node.manufacturerId === undefined) {
@@ -456,7 +455,7 @@ function onNodeInterviewCompleted (zwaveNode) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').NodeInterviewFailedEventArgs} args
  */
-function onNodeInterviewFailed (zwaveNode, args) {
+function onNodeInterviewFailed(zwaveNode, args) {
   logger.error(
     `Interview of node ${zwaveNode.id} has failed: ${args.errorMessage}`
   )
@@ -477,10 +476,9 @@ function onNodeInterviewFailed (zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {NodeStatus} oldStatus
  */
-function onNodeWakeUp (zwaveNode, oldStatus) {
+function onNodeWakeUp(zwaveNode, oldStatus) {
   logger.info(
-    `Node ${zwaveNode.id} is ${
-      oldStatus === NodeStatus.Unknown ? '' : 'now '
+    `Node ${zwaveNode.id} is ${oldStatus === NodeStatus.Unknown ? '' : 'now '
     }awake`
   )
 
@@ -499,10 +497,9 @@ function onNodeWakeUp (zwaveNode, oldStatus) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {NodeStatus} oldStatus
  */
-function onNodeSleep (zwaveNode, oldStatus) {
+function onNodeSleep(zwaveNode, oldStatus) {
   logger.info(
-    `Node ${zwaveNode.id} is ${
-      oldStatus === NodeStatus.Unknown ? '' : 'now '
+    `Node ${zwaveNode.id} is ${oldStatus === NodeStatus.Unknown ? '' : 'now '
     }asleep`
   )
   onNodeStatus.call(this, zwaveNode)
@@ -520,7 +517,7 @@ function onNodeSleep (zwaveNode, oldStatus) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {NodeStatus} oldStatus
  */
-function onNodeAlive (zwaveNode, oldStatus) {
+function onNodeAlive(zwaveNode, oldStatus) {
   onNodeStatus.call(this, zwaveNode)
   if (oldStatus === NodeStatus.Dead) {
     logger.info(`Node ${zwaveNode.id}: has returned from the dead`)
@@ -542,11 +539,10 @@ function onNodeAlive (zwaveNode, oldStatus) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {NodeStatus} oldStatus
  */
-function onNodeDead (zwaveNode, oldStatus) {
+function onNodeDead(zwaveNode, oldStatus) {
   onNodeStatus.call(this, zwaveNode)
   logger.info(
-    `Node ${zwaveNode.id} is ${
-      oldStatus === NodeStatus.Unknown ? '' : 'now '
+    `Node ${zwaveNode.id} is ${oldStatus === NodeStatus.Unknown ? '' : 'now '
     }dead`
   )
 
@@ -564,7 +560,7 @@ function onNodeDead (zwaveNode, oldStatus) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueAddedArgs} args
  */
-function onNodeValueAdded (zwaveNode, args) {
+function onNodeValueAdded(zwaveNode, args) {
   logger.info(
     `Node ${zwaveNode.id}: value added: ${getValueID(args)} => ${args.newValue}`
   )
@@ -589,7 +585,7 @@ function onNodeValueAdded (zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueNotificationArgs} args
  */
-function onNodeValueNotification (zwaveNode, args) {
+function onNodeValueNotification(zwaveNode, args) {
   // notification hasn't `newValue`
   args.newValue = args.value
   // specify that this is stateless
@@ -604,13 +600,11 @@ function onNodeValueNotification (zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNodeValueUpdatedArgs} args
  * @param {boolean} isNotification
  */
-function onNodeValueUpdated (zwaveNode, args) {
+function onNodeValueUpdated(zwaveNode, args) {
   updateValue.call(this, zwaveNode, args)
   logger.info(
-    `Node ${zwaveNode.id}: value ${
-      args.stateless ? 'notification' : 'updated'
-    }: ${getValueID(args)} ${
-      args.stateless ? args.newValue : `${args.prevValue} => ${args.newValue}`
+    `Node ${zwaveNode.id}: value ${args.stateless ? 'notification' : 'updated'
+    }: ${getValueID(args)} ${args.stateless ? args.newValue : `${args.prevValue} => ${args.newValue}`
     }`
   )
 
@@ -629,7 +623,7 @@ function onNodeValueUpdated (zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueRemovedArgs} args
  */
-function onNodeValueRemoved (zwaveNode, args) {
+function onNodeValueRemoved(zwaveNode, args) {
   removeValue.call(this, zwaveNode, args)
   logger.info(`Node ${zwaveNode.id}: value removed: ${getValueID(args)}`)
   this.emit(
@@ -647,7 +641,7 @@ function onNodeValueRemoved (zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeMetadataUpdatedArgs} args
  */
-function onNodeMetadataUpdated (zwaveNode, args) {
+function onNodeMetadataUpdated(zwaveNode, args) {
   const valueId = parseValue.call(this, zwaveNode, args, args.metadata)
   logger.info(`Node ${valueId.nodeId}: metadata updated: ${getValueID(args)}`)
   this.emit(
@@ -666,7 +660,7 @@ function onNodeMetadataUpdated (zwaveNode, args) {
  * @param {CommandClasses} ccId
  * @param {Record<string, unknown>} args
  */
-function onNodeNotification (zwaveNode, ccId, args) {
+function onNodeNotification(zwaveNode, ccId, args) {
   const valueId = {
     id: null,
     nodeId: zwaveNode.id,
@@ -702,6 +696,7 @@ function onNodeNotification (zwaveNode, ccId, args) {
   }
 
   valueId.id = getValueID(valueId, true)
+  valueId.propertyName = valueId.property // must be defined in named topics
 
   logger.log(
     'info',
@@ -732,7 +727,7 @@ function onNodeNotification (zwaveNode, ccId, args) {
  * @param {number} sentFragments
  * @param {number} totalFragments
  */
-function onNodeFirmwareUpdateProgress (
+function onNodeFirmwareUpdateProgress(
   zwaveNode,
   sentFragments,
   totalFragments
@@ -758,7 +753,7 @@ function onNodeFirmwareUpdateProgress (
  * @param {FirmwareUpdateStatus} status
  * @param {number} waitTime
  */
-function onNodeFirmwareUpdateFinished (zwaveNode, status, waitTime) {
+function onNodeFirmwareUpdateFinished(zwaveNode, status, waitTime) {
   updateControllerStatus.call(
     this,
     `Node ${zwaveNode.id} firmware update FINISHED: Status ${status}, Time: ${waitTime}`
@@ -781,7 +776,7 @@ function onNodeFirmwareUpdateFinished (zwaveNode, status, waitTime) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function bindNodeEvents (zwaveNode) {
+function bindNodeEvents(zwaveNode) {
   logger.debug(`Binding to node ${zwaveNode.id} events`)
 
   // https://zwave-js.github.io/node-zwave-js/#/api/node?id=zwavenode-events
@@ -808,7 +803,7 @@ function bindNodeEvents (zwaveNode) {
  *
  * @param {number} nodeid
  */
-function removeNode (nodeid) {
+function removeNode(nodeid) {
   logger.info(`Node removed ${nodeid}`)
 
   // don't use splice here, nodeid equals to the index in the array
@@ -827,7 +822,7 @@ function removeNode (nodeid) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @returns {import('../types/index.js').Z2MNode}
  */
-function addNode (zwaveNode) {
+function addNode(zwaveNode) {
   const nodeId = zwaveNode.id
 
   let node = this.nodes.get(nodeId)
@@ -887,7 +882,7 @@ function addNode (zwaveNode) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function dumpNode (zwaveNode) {
+function dumpNode(zwaveNode) {
   const nodeId = zwaveNode.id
 
   const node = this.nodes.get(nodeId)
@@ -898,9 +893,8 @@ function dumpNode (zwaveNode) {
     utils.num2hex(zwaveNode.productType)
   ]
   node.hexId = `${hexIds[0]}-${hexIds[2]}-${hexIds[1]}`
-  node.dbLink = `https://devices.zwave-js.io/?jumpTo=${hexIds[0]}:${
-    hexIds[2]
-  }:${hexIds[1]}:${node.firmwareVersion || '0.0'}`
+  node.dbLink = `https://devices.zwave-js.io/?jumpTo=${hexIds[0]}:${hexIds[2]
+    }:${hexIds[1]}:${node.firmwareVersion || '0.0'}`
 
   const deviceConfig = zwaveNode.deviceConfig || {
     label: `Unknown product ${hexIds[1]}`,
@@ -979,7 +973,7 @@ function dumpNode (zwaveNode) {
  * @param {import('zwave-js').ValueMetadata} zwaveValueMeta
  * @returns The internal valueId
  */
-function updateValueMetadata (zwaveNode, zwaveValue, zwaveValueMeta) {
+function updateValueMetadata(zwaveNode, zwaveValue, zwaveValueMeta) {
   zwaveValue.nodeId = zwaveNode.id
 
   const valueId = {
@@ -1040,7 +1034,7 @@ function updateValueMetadata (zwaveNode, zwaveValue, zwaveValueMeta) {
  * @param { import('zwave-js').TranslatedValueID } zwaveValue
  * @param {Map<string, import('../types/index.js').Z2MValueId} oldValues old valueIds, used to check if the value was existing or not
  */
-function addValue (zwaveNode, zwaveValue, oldValues) {
+function addValue(zwaveNode, zwaveValue, oldValues) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (!node) {
@@ -1072,7 +1066,7 @@ function addValue (zwaveNode, zwaveValue, oldValues) {
  * @param { import('zwave-js').ZWaveNode } zwaveNode
  * @param { import('zwave-js').TranslatedValueID } zwaveValue
  */
-function parseValue (zwaveNode, zwaveValue, zwaveValueMeta) {
+function parseValue(zwaveNode, zwaveValue, zwaveValueMeta) {
   const node = this.nodes.get(zwaveNode.id)
   const valueId = updateValueMetadata.call(
     this,
@@ -1115,7 +1109,7 @@ function parseValue (zwaveNode, zwaveValue, zwaveValueMeta) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueUpdatedArgs} args
  */
-function updateValue (zwaveNode, args) {
+function updateValue(zwaveNode, args) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (!node) {
@@ -1168,7 +1162,7 @@ function updateValue (zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueRemovedArgs} args
  */
-function removeValue (zwaveNode, args) {
+function removeValue(zwaveNode, args) {
   const node = this.nodes.get(zwaveNode.id)
   const vID = getValueID(args)
   const toRemove = node ? node.values[vID] : null
@@ -1184,7 +1178,7 @@ function removeValue (zwaveNode, args) {
 
 // ------- Utils ------------------------
 
-function parseNotification (parameters) {
+function parseNotification(parameters) {
   if (Buffer.isBuffer(parameters)) {
     return parameters.toString('hex')
   } else if (parameters instanceof Duration) {
@@ -1200,7 +1194,7 @@ function parseNotification (parameters) {
  * @param {import('../types/index.js').Z2MNode} node Internal node object
  * @returns A string in the format `<manufacturerId>-<productId>-<producttype>` that unique identifhy a zwave device
  */
-function getDeviceID (node) {
+function getDeviceID(node) {
   if (!node) return ''
 
   return `${parseInt(node.manufacturerId)}-${parseInt(
@@ -1214,7 +1208,7 @@ function getDeviceID (node) {
  * @param {import('zwave-js').TranslatedValueID} valueId
  * @returns true if this is a current value, false otherwise
  */
-function isCurrentValue (valueId) {
+function isCurrentValue(valueId) {
   return valueId.propertyName && /current/i.test(valueId.propertyName)
 }
 
@@ -1225,7 +1219,7 @@ function isCurrentValue (valueId) {
  * @param {TranslatedValueID[]} definedValueIds
  * @returns
  */
-function findTargetValue (zwaveValue, definedValueIds) {
+function findTargetValue(zwaveValue, definedValueIds) {
   return definedValueIds.find(
     v =>
       v.commandClass === zwaveValue.commandClass &&
@@ -1242,7 +1236,7 @@ function findTargetValue (zwaveValue, definedValueIds) {
  * @param {boolean} withNode Add node identifier
  * @returns The value id unique identifier
  */
-function getValueID (v, withNode) {
+function getValueID(v, withNode) {
   return `${withNode ? v.nodeId + '-' : ''}${v.commandClass}-${v.endpoint ||
     0}-${v.property}${v.propertyKey !== undefined ? '-' + v.propertyKey : ''}`
 }
@@ -1253,7 +1247,7 @@ function getValueID (v, withNode) {
  * context - what you want "this" to be.
  * params - array of parameters to pass to function.
  */
-function wrapFunction (fn, context, params) {
+function wrapFunction(fn, context, params) {
   return function () {
     fn.apply(context, params)
   }
@@ -1265,7 +1259,7 @@ function wrapFunction (fn, context, params) {
  * @param {*} obj The object to copy
  * @returns The copied object
  */
-function copy (obj) {
+function copy(obj) {
   return JSON.parse(JSON.stringify(obj))
 }
 
@@ -2133,7 +2127,7 @@ ZwaveClient.prototype.setPollInterval = function (valueId, interval) {
  * @param {import('../types/index.js').Z2MValueId} valueId
  * @param {number} interval seconds
  */
-async function tryPoll (valueId, interval) {
+async function tryPoll(valueId, interval) {
   try {
     await this.pollValue(valueId)
   } catch (error) {
@@ -2561,8 +2555,7 @@ ZwaveClient.prototype.writeBroadcast = async function (valueId, value) {
       await broadcastNode.setValue(valueId, value)
     } catch (error) {
       logger.error(
-        `Error while sending broadcast ${value} to CC ${valueId.commandClass} ${
-          valueId.property
+        `Error while sending broadcast ${value} to CC ${valueId.commandClass} ${valueId.property
         } ${valueId.propertyKey || ''}: ${error.message}`
       )
     }
@@ -2585,8 +2578,7 @@ ZwaveClient.prototype.writeMulticast = async function (nodes, valueId, value) {
     } catch (error) {
       fallback = error.code === ZWaveErrorCodes.CC_NotSupported
       logger.error(
-        `Error while sending multicast ${value} to CC ${valueId.commandClass} ${
-          valueId.property
+        `Error while sending multicast ${value} to CC ${valueId.commandClass} ${valueId.property
         } ${valueId.propertyKey || ''}: ${error.message}`
       )
     }

--- a/lib/ZwaveClient.js
+++ b/lib/ZwaveClient.js
@@ -90,7 +90,7 @@ const ZWAVEJS_LOG_FILE = utils.joinPath(storeDir, `zwavejs_${process.pid}.log`)
  * @param {Socket} socket
  * @returns {import('../types/index.js').ZwaveClient}
  */
-function ZwaveClient(config, socket) {
+function ZwaveClient (config, socket) {
   if (!(this instanceof ZwaveClient)) {
     return new ZwaveClient(config)
   }
@@ -140,7 +140,7 @@ inherits(ZwaveClient, EventEmitter)
 
 // ---------- DRIVER EVENTS -------------------------------------
 
-function driverReady() {
+function driverReady () {
   /*
     Now the controller interview is complete. This means we know which nodes
     are included in the network, but they might not be ready yet.
@@ -194,7 +194,7 @@ function driverReady() {
   logger.info(`Scanning network with homeid: ${homeHex}`)
 }
 
-function driverError(error) {
+function driverError (error) {
   this.error = 'Driver: ' + error.message
   this.status = ZWAVE_STATUS.driverFailed
   updateControllerStatus.call(this, this.error)
@@ -202,7 +202,7 @@ function driverError(error) {
   this.emit('event', eventEmitter.driver, 'driver error', error)
 }
 
-function scanComplete() {
+function scanComplete () {
   this.scanComplete = true
 
   updateControllerStatus.call(this, 'Scan completed')
@@ -219,43 +219,43 @@ function scanComplete() {
 
 // ---------- CONTROLLER EVENTS -------------------------------
 
-function updateControllerStatus(status) {
+function updateControllerStatus (status) {
   logger.info(`Controller status: ${status}`)
   this.cntStatus = status
   this.sendToSocket(socketEvents.controller, status)
 }
 
-function onInclusionStarted(secure) {
+function onInclusionStarted (secure) {
   const message = `${secure ? 'Secure' : 'Non-secure'} inclusion started`
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'inclusion started', secure)
 }
 
-function onExclusionStarted() {
+function onExclusionStarted () {
   const message = 'Exclusion started'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'exclusion started')
 }
 
-function onInclusionStopped() {
+function onInclusionStopped () {
   const message = 'Inclusion stopped'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'inclusion stopped')
 }
 
-function onExclusionStopped() {
+function onExclusionStopped () {
   const message = 'Exclusion stopped'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'exclusion stopped')
 }
 
-function onInclusionFailed() {
+function onInclusionFailed () {
   const message = 'Inclusion failed'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'inclusion failed')
 }
 
-function onExclusionFailed() {
+function onExclusionFailed () {
   const message = 'Exclusion failed'
   updateControllerStatus.call(this, message)
   this.emit('event', eventEmitter.controller, 'exclusion failed')
@@ -266,7 +266,7 @@ function onExclusionFailed() {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeAdded(zwaveNode) {
+function onNodeAdded (zwaveNode) {
   logger.info(`Node ${zwaveNode.id}: added`)
 
   // the driver is ready so this node has been added on fly
@@ -288,7 +288,7 @@ function onNodeAdded(zwaveNode) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeRemoved(zwaveNode) {
+function onNodeRemoved (zwaveNode) {
   logger.info(`Node ${zwaveNode.id}: removed`)
   zwaveNode.removeAllListeners()
 
@@ -307,7 +307,7 @@ function onNodeRemoved(zwaveNode) {
  *
  * @param {ReadonlyMap<number, import('zwave-js').HealNodeStatus>} progress
  */
-function onHealNetworkProgress(progress) {
+function onHealNetworkProgress (progress) {
   const toHeal = [...progress.values()]
   const healedNodes = toHeal.filter(v => v !== 'pending')
   let message
@@ -335,7 +335,7 @@ function onHealNetworkProgress(progress) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeStatus(zwaveNode) {
+function onNodeStatus (zwaveNode) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (node) {
@@ -356,7 +356,7 @@ function onNodeStatus(zwaveNode) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeReady(zwaveNode) {
+function onNodeReady (zwaveNode) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (!node) {
@@ -417,7 +417,8 @@ function onNodeReady(zwaveNode) {
   )
 
   logger.info(
-    `Node ${node.id} ready: ${node.manufacturer} - ${node.productLabel
+    `Node ${node.id} ready: ${node.manufacturer} - ${
+      node.productLabel
     } (${node.productDescription || 'Unknown'})`
   )
 }
@@ -428,7 +429,7 @@ function onNodeReady(zwaveNode) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function onNodeInterviewCompleted(zwaveNode) {
+function onNodeInterviewCompleted (zwaveNode) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (node.manufacturerId === undefined) {
@@ -455,7 +456,7 @@ function onNodeInterviewCompleted(zwaveNode) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').NodeInterviewFailedEventArgs} args
  */
-function onNodeInterviewFailed(zwaveNode, args) {
+function onNodeInterviewFailed (zwaveNode, args) {
   logger.error(
     `Interview of node ${zwaveNode.id} has failed: ${args.errorMessage}`
   )
@@ -476,9 +477,10 @@ function onNodeInterviewFailed(zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {NodeStatus} oldStatus
  */
-function onNodeWakeUp(zwaveNode, oldStatus) {
+function onNodeWakeUp (zwaveNode, oldStatus) {
   logger.info(
-    `Node ${zwaveNode.id} is ${oldStatus === NodeStatus.Unknown ? '' : 'now '
+    `Node ${zwaveNode.id} is ${
+      oldStatus === NodeStatus.Unknown ? '' : 'now '
     }awake`
   )
 
@@ -497,9 +499,10 @@ function onNodeWakeUp(zwaveNode, oldStatus) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {NodeStatus} oldStatus
  */
-function onNodeSleep(zwaveNode, oldStatus) {
+function onNodeSleep (zwaveNode, oldStatus) {
   logger.info(
-    `Node ${zwaveNode.id} is ${oldStatus === NodeStatus.Unknown ? '' : 'now '
+    `Node ${zwaveNode.id} is ${
+      oldStatus === NodeStatus.Unknown ? '' : 'now '
     }asleep`
   )
   onNodeStatus.call(this, zwaveNode)
@@ -517,7 +520,7 @@ function onNodeSleep(zwaveNode, oldStatus) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {NodeStatus} oldStatus
  */
-function onNodeAlive(zwaveNode, oldStatus) {
+function onNodeAlive (zwaveNode, oldStatus) {
   onNodeStatus.call(this, zwaveNode)
   if (oldStatus === NodeStatus.Dead) {
     logger.info(`Node ${zwaveNode.id}: has returned from the dead`)
@@ -539,10 +542,11 @@ function onNodeAlive(zwaveNode, oldStatus) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {NodeStatus} oldStatus
  */
-function onNodeDead(zwaveNode, oldStatus) {
+function onNodeDead (zwaveNode, oldStatus) {
   onNodeStatus.call(this, zwaveNode)
   logger.info(
-    `Node ${zwaveNode.id} is ${oldStatus === NodeStatus.Unknown ? '' : 'now '
+    `Node ${zwaveNode.id} is ${
+      oldStatus === NodeStatus.Unknown ? '' : 'now '
     }dead`
   )
 
@@ -560,7 +564,7 @@ function onNodeDead(zwaveNode, oldStatus) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueAddedArgs} args
  */
-function onNodeValueAdded(zwaveNode, args) {
+function onNodeValueAdded (zwaveNode, args) {
   logger.info(
     `Node ${zwaveNode.id}: value added: ${getValueID(args)} => ${args.newValue}`
   )
@@ -585,7 +589,7 @@ function onNodeValueAdded(zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueNotificationArgs} args
  */
-function onNodeValueNotification(zwaveNode, args) {
+function onNodeValueNotification (zwaveNode, args) {
   // notification hasn't `newValue`
   args.newValue = args.value
   // specify that this is stateless
@@ -600,11 +604,13 @@ function onNodeValueNotification(zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNodeValueUpdatedArgs} args
  * @param {boolean} isNotification
  */
-function onNodeValueUpdated(zwaveNode, args) {
+function onNodeValueUpdated (zwaveNode, args) {
   updateValue.call(this, zwaveNode, args)
   logger.info(
-    `Node ${zwaveNode.id}: value ${args.stateless ? 'notification' : 'updated'
-    }: ${getValueID(args)} ${args.stateless ? args.newValue : `${args.prevValue} => ${args.newValue}`
+    `Node ${zwaveNode.id}: value ${
+      args.stateless ? 'notification' : 'updated'
+    }: ${getValueID(args)} ${
+      args.stateless ? args.newValue : `${args.prevValue} => ${args.newValue}`
     }`
   )
 
@@ -623,7 +629,7 @@ function onNodeValueUpdated(zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueRemovedArgs} args
  */
-function onNodeValueRemoved(zwaveNode, args) {
+function onNodeValueRemoved (zwaveNode, args) {
   removeValue.call(this, zwaveNode, args)
   logger.info(`Node ${zwaveNode.id}: value removed: ${getValueID(args)}`)
   this.emit(
@@ -641,7 +647,7 @@ function onNodeValueRemoved(zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeMetadataUpdatedArgs} args
  */
-function onNodeMetadataUpdated(zwaveNode, args) {
+function onNodeMetadataUpdated (zwaveNode, args) {
   const valueId = parseValue.call(this, zwaveNode, args, args.metadata)
   logger.info(`Node ${valueId.nodeId}: metadata updated: ${getValueID(args)}`)
   this.emit(
@@ -660,7 +666,7 @@ function onNodeMetadataUpdated(zwaveNode, args) {
  * @param {CommandClasses} ccId
  * @param {Record<string, unknown>} args
  */
-function onNodeNotification(zwaveNode, ccId, args) {
+function onNodeNotification (zwaveNode, ccId, args) {
   const valueId = {
     id: null,
     nodeId: zwaveNode.id,
@@ -727,7 +733,7 @@ function onNodeNotification(zwaveNode, ccId, args) {
  * @param {number} sentFragments
  * @param {number} totalFragments
  */
-function onNodeFirmwareUpdateProgress(
+function onNodeFirmwareUpdateProgress (
   zwaveNode,
   sentFragments,
   totalFragments
@@ -753,7 +759,7 @@ function onNodeFirmwareUpdateProgress(
  * @param {FirmwareUpdateStatus} status
  * @param {number} waitTime
  */
-function onNodeFirmwareUpdateFinished(zwaveNode, status, waitTime) {
+function onNodeFirmwareUpdateFinished (zwaveNode, status, waitTime) {
   updateControllerStatus.call(
     this,
     `Node ${zwaveNode.id} firmware update FINISHED: Status ${status}, Time: ${waitTime}`
@@ -776,7 +782,7 @@ function onNodeFirmwareUpdateFinished(zwaveNode, status, waitTime) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function bindNodeEvents(zwaveNode) {
+function bindNodeEvents (zwaveNode) {
   logger.debug(`Binding to node ${zwaveNode.id} events`)
 
   // https://zwave-js.github.io/node-zwave-js/#/api/node?id=zwavenode-events
@@ -803,7 +809,7 @@ function bindNodeEvents(zwaveNode) {
  *
  * @param {number} nodeid
  */
-function removeNode(nodeid) {
+function removeNode (nodeid) {
   logger.info(`Node removed ${nodeid}`)
 
   // don't use splice here, nodeid equals to the index in the array
@@ -822,7 +828,7 @@ function removeNode(nodeid) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @returns {import('../types/index.js').Z2MNode}
  */
-function addNode(zwaveNode) {
+function addNode (zwaveNode) {
   const nodeId = zwaveNode.id
 
   let node = this.nodes.get(nodeId)
@@ -882,7 +888,7 @@ function addNode(zwaveNode) {
  *
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  */
-function dumpNode(zwaveNode) {
+function dumpNode (zwaveNode) {
   const nodeId = zwaveNode.id
 
   const node = this.nodes.get(nodeId)
@@ -893,8 +899,9 @@ function dumpNode(zwaveNode) {
     utils.num2hex(zwaveNode.productType)
   ]
   node.hexId = `${hexIds[0]}-${hexIds[2]}-${hexIds[1]}`
-  node.dbLink = `https://devices.zwave-js.io/?jumpTo=${hexIds[0]}:${hexIds[2]
-    }:${hexIds[1]}:${node.firmwareVersion || '0.0'}`
+  node.dbLink = `https://devices.zwave-js.io/?jumpTo=${hexIds[0]}:${
+    hexIds[2]
+  }:${hexIds[1]}:${node.firmwareVersion || '0.0'}`
 
   const deviceConfig = zwaveNode.deviceConfig || {
     label: `Unknown product ${hexIds[1]}`,
@@ -973,7 +980,7 @@ function dumpNode(zwaveNode) {
  * @param {import('zwave-js').ValueMetadata} zwaveValueMeta
  * @returns The internal valueId
  */
-function updateValueMetadata(zwaveNode, zwaveValue, zwaveValueMeta) {
+function updateValueMetadata (zwaveNode, zwaveValue, zwaveValueMeta) {
   zwaveValue.nodeId = zwaveNode.id
 
   const valueId = {
@@ -1034,7 +1041,7 @@ function updateValueMetadata(zwaveNode, zwaveValue, zwaveValueMeta) {
  * @param { import('zwave-js').TranslatedValueID } zwaveValue
  * @param {Map<string, import('../types/index.js').Z2MValueId} oldValues old valueIds, used to check if the value was existing or not
  */
-function addValue(zwaveNode, zwaveValue, oldValues) {
+function addValue (zwaveNode, zwaveValue, oldValues) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (!node) {
@@ -1066,7 +1073,7 @@ function addValue(zwaveNode, zwaveValue, oldValues) {
  * @param { import('zwave-js').ZWaveNode } zwaveNode
  * @param { import('zwave-js').TranslatedValueID } zwaveValue
  */
-function parseValue(zwaveNode, zwaveValue, zwaveValueMeta) {
+function parseValue (zwaveNode, zwaveValue, zwaveValueMeta) {
   const node = this.nodes.get(zwaveNode.id)
   const valueId = updateValueMetadata.call(
     this,
@@ -1109,7 +1116,7 @@ function parseValue(zwaveNode, zwaveValue, zwaveValueMeta) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueUpdatedArgs} args
  */
-function updateValue(zwaveNode, args) {
+function updateValue (zwaveNode, args) {
   const node = this.nodes.get(zwaveNode.id)
 
   if (!node) {
@@ -1162,7 +1169,7 @@ function updateValue(zwaveNode, args) {
  * @param {import('zwave-js').ZWaveNode} zwaveNode
  * @param {import('zwave-js').ZWaveNodeValueRemovedArgs} args
  */
-function removeValue(zwaveNode, args) {
+function removeValue (zwaveNode, args) {
   const node = this.nodes.get(zwaveNode.id)
   const vID = getValueID(args)
   const toRemove = node ? node.values[vID] : null
@@ -1178,7 +1185,7 @@ function removeValue(zwaveNode, args) {
 
 // ------- Utils ------------------------
 
-function parseNotification(parameters) {
+function parseNotification (parameters) {
   if (Buffer.isBuffer(parameters)) {
     return parameters.toString('hex')
   } else if (parameters instanceof Duration) {
@@ -1194,7 +1201,7 @@ function parseNotification(parameters) {
  * @param {import('../types/index.js').Z2MNode} node Internal node object
  * @returns A string in the format `<manufacturerId>-<productId>-<producttype>` that unique identifhy a zwave device
  */
-function getDeviceID(node) {
+function getDeviceID (node) {
   if (!node) return ''
 
   return `${parseInt(node.manufacturerId)}-${parseInt(
@@ -1208,7 +1215,7 @@ function getDeviceID(node) {
  * @param {import('zwave-js').TranslatedValueID} valueId
  * @returns true if this is a current value, false otherwise
  */
-function isCurrentValue(valueId) {
+function isCurrentValue (valueId) {
   return valueId.propertyName && /current/i.test(valueId.propertyName)
 }
 
@@ -1219,7 +1226,7 @@ function isCurrentValue(valueId) {
  * @param {TranslatedValueID[]} definedValueIds
  * @returns
  */
-function findTargetValue(zwaveValue, definedValueIds) {
+function findTargetValue (zwaveValue, definedValueIds) {
   return definedValueIds.find(
     v =>
       v.commandClass === zwaveValue.commandClass &&
@@ -1236,7 +1243,7 @@ function findTargetValue(zwaveValue, definedValueIds) {
  * @param {boolean} withNode Add node identifier
  * @returns The value id unique identifier
  */
-function getValueID(v, withNode) {
+function getValueID (v, withNode) {
   return `${withNode ? v.nodeId + '-' : ''}${v.commandClass}-${v.endpoint ||
     0}-${v.property}${v.propertyKey !== undefined ? '-' + v.propertyKey : ''}`
 }
@@ -1247,7 +1254,7 @@ function getValueID(v, withNode) {
  * context - what you want "this" to be.
  * params - array of parameters to pass to function.
  */
-function wrapFunction(fn, context, params) {
+function wrapFunction (fn, context, params) {
   return function () {
     fn.apply(context, params)
   }
@@ -1259,7 +1266,7 @@ function wrapFunction(fn, context, params) {
  * @param {*} obj The object to copy
  * @returns The copied object
  */
-function copy(obj) {
+function copy (obj) {
   return JSON.parse(JSON.stringify(obj))
 }
 
@@ -2127,7 +2134,7 @@ ZwaveClient.prototype.setPollInterval = function (valueId, interval) {
  * @param {import('../types/index.js').Z2MValueId} valueId
  * @param {number} interval seconds
  */
-async function tryPoll(valueId, interval) {
+async function tryPoll (valueId, interval) {
   try {
     await this.pollValue(valueId)
   } catch (error) {
@@ -2555,7 +2562,8 @@ ZwaveClient.prototype.writeBroadcast = async function (valueId, value) {
       await broadcastNode.setValue(valueId, value)
     } catch (error) {
       logger.error(
-        `Error while sending broadcast ${value} to CC ${valueId.commandClass} ${valueId.property
+        `Error while sending broadcast ${value} to CC ${valueId.commandClass} ${
+          valueId.property
         } ${valueId.propertyKey || ''}: ${error.message}`
       )
     }
@@ -2578,7 +2586,8 @@ ZwaveClient.prototype.writeMulticast = async function (nodes, valueId, value) {
     } catch (error) {
       fallback = error.code === ZWaveErrorCodes.CC_NotSupported
       logger.error(
-        `Error while sending multicast ${value} to CC ${valueId.commandClass} ${valueId.property
+        `Error while sending multicast ${value} to CC ${valueId.commandClass} ${
+          valueId.property
         } ${valueId.propertyKey || ''}: ${error.message}`
       )
     }

